parquet/sandboxes/heka/output/s3_parquet.lua

```diff
@@ -242,6 +246,9 @@ local function get_s3_path(json)
         else
             v = read_json(json, d.source) or default_nil
         end
+        if d.dateformat and type(v) == "number" then
```

Comment:
So this allows any field to contain dateformat but requires it is in nanoseconds.  We could either restrict it to Timestamp only and dump the number test or we allow a scaling factor (say there is a time_t in some other field)